frontend/src/app/app.component.ts

```diff
@@ -46,4 +52,10 @@ export class AppComponent implements OnInit {
     }
     return false;
   }
+  shortUrl(){
+    if (this.url.length<39){
```

Comment:
Also, let's declare a constant or environment variable with value `39` and use it here

Reply:
okay